src/ralph/admin/autocomplete.py

```diff
@@ -92,25 +96,35 @@ def get_urls(self):
         outer_model = self.model
 
         class Detail(SuggestView):
+            empty_value = AUTOCOMPLETE_EMPTY_VALUE
             model = outer_model
 
             def dispatch(self, request, *args, **kwargs):
                 self.pk = request.GET.get(DETAIL_PARAM, None)
                 if not self.pk:
                     return HttpResponseBadRequest()
-                self.pks = self.pk.split(',')
+                self.values = list(map(int, self.pk.split(',')))
```

Comment:
i'm not sure if mapping to int is good idea here - what if we'll have uid somewhere in the future?